<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="py_pubsub" exec="sensor_node" name="sensor_2_publisher">
    <param name="sensor_data" value="2" />
    <param name="publish_rate" value="2" />
  </node>
  <node pkg="py_pubsub" exec="sensor_node" name="sensor_1_publisher" >
    <param name="publish_rate" value="2" />
    </node>
  <node pkg="py_pubsub" exec="sensor_listener" />
</launch>